azkaban-common/src/main/java/azkaban/executor/Executor.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2014 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.executor;
+
+/**
+ * Class to represent an AzkabanExecutorServer details for ExecutorManager
+ *
+ * @author gaggarwa
+ */
+public class Executor {
+  private final int id;
+  private String host;
+  private int port;
+  private boolean active = true;
+
+  // TODO: ExecutorStats to be added
+
+  public Executor(int id) {
+    this.id = id;
+  }
+
+  public Executor(int id, String host, int port) {
+    this.id = id;
```

Comment:
Should we do basic validations on these constructor parameters?
port can't < 0
host can't be null
id can't < 0